packages/gateway/src/features/registration/root-resolvers.ts

```diff
@@ -533,7 +538,7 @@ export const resolvers: GQLResolver = {
         throw new UnassignError('User has been unassigned')
       }
       if (hasScope(authHeader, 'register')) {
-        return markEventAsRegistered(
+        return await markEventAsRegistered(
```

Comment:
Did you notice a difference when this wasn't here? I'm thinking there should be need to await here as there's no try-catch wrapping this. End result for the caller is a promise regardless.

```suggestion
        return markEventAsRegistered(
```